shelf_multipart/pubspec.yaml

```
@@ -9,10 +9,10 @@ environment:
 
 dependencies:
   http_parser: ^4.0.0
-  mime: ^1.0.0
+  mime: ^2.0.0
```

Given that we don't appear to be affected by the breaking change, can we make this

```suggestion
  mime: '>=1.0.0 <3.0.0'
```